contracts/src/core/EigenDAAccessControl.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {AccessControl} from "@openzeppelin/contracts/access/AccessControl.sol";
+import {AccessControlConstants} from "src/core/libraries/v3/access-control/AccessControlConstants.sol";
+
+contract EigenDAAccessControl is AccessControl {
+    constructor(address owner) {
+        _setupRole(AccessControlConstants.OWNER_ROLE, owner);
```

Comment:
which version of OZ are we on? seems like _setupRole is deprecated in favor of _grantRole in v4.9.6: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/dc44c9f1a4c3b10af99492eed84f83ed244203f6/contracts/access/AccessControl.sol#L204

Reply:
https://github.com/Layr-Labs/eigenda/pull/1758/commits/fcf8dadb1063baed05334720fe7370cdc964c9f9